configure.ac

```diff
@@ -49,6 +48,18 @@ AC_ARG_WITH([module-prefix],
             [],
             [with_module_prefix=$prefix])
 
+# Check for gtest being requested
+AC_ARG_ENABLE([gtest],
+  [AS_HELP_STRING([--enable-gtest],
+                  [Enable tests using the Google C++ Testing Framework.
+                  (Default is disabled.)])],
+  [enable_gtest=$enableval],
```

Comment:
IMO should fail configure if enable_gtest=yes and have_cxx14=no

Reply:
fixed, wrapped the c++14 check to run only when gtest is enabled and simply fail if not met